pkg/util/intstr/intstr.go

```diff
@@ -67,7 +71,17 @@ func (intstr *IntOrString) String() string {
 	if intstr.Type == String {
 		return intstr.StrVal
 	}
-	return strconv.Itoa(intstr.IntVal)
+	return strconv.Itoa(intstr.IntValue())
+}
+
+// IntValue returns the IntVal if type Int, or if
+// it is a String, will attempt a conversion to int.
+func (intstr *IntOrString) IntValue() int {
+	if intstr.Type == String {
```

Comment:
I think this is wrong and we should not do it.  It's scope creep for this PR anyway.  Was there a use case justifying this?  I think we should roll it back